VERSION

```diff
@@ -1 +1 @@
-0.0.2
+0.0.3a
```

Comment:
Do we want to merge with `3a`?

Reply:
I updated it here, https://github.com/woven-planet/wicker/pull/33